keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -723,22 +828,17 @@ _This section must be completed when targeting alpha to a release._
       of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
 
 * **Does enabling the feature change any default behavior?**
-  Any change of default behavior may be surprising to users or break existing
-  automations, so be extremely careful here.
+  No
 
 * **Can the feature be disabled once it has been enabled (i.e. can we roll back
   the enablement)?**
-  Also set `disable-supported` to `true` or `false` in `kep.yaml`.
-  Describe the consequences on existing workloads (e.g., if this is a runtime
-  feature, can it break the existing applications?).
+  This feature can be disabled. If this feature is disabled then newly created privileged Windows containers run as non-privileged containers and will likely encounter errors.
```

Comment:
is this true even if the existing field is re-used?  I'd like to have a stronger statement about what we expect.  "likely encounter errors" is a little weak. Do we expect to reject the pod at admission time somehow?  Will the pod be run as non-privileged (that seems not-awesome).

Reply:
This would be true even if the existing field is re-used.
I'll update this section. We're planning on having the container runtime (contianerd) fail to start these privileged containers if flags aren't passed in the CRI calls.

Reply:
@deads2k are my updates here OK?